src/source/Metrics/Metrics.c

```diff
@@ -140,6 +140,26 @@ STATUS getRtpInboundStats(PRtcPeerConnection pRtcPeerConnection, PRtcRtpTranscei
     return retStatus;
 }
 
+STATUS getDataChannelStats(PRtcPeerConnection pRtcPeerConnection, PRtcDataChannelStats pRtcDataChannelStats)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    PKvsDataChannel pKvsDataChannel = NULL;
+    UINT32 numberOfDataChannels = 0;
+    PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
+    CHK(pRtcPeerConnection != NULL || pRtcDataChannelStats != NULL, STATUS_NULL_ARG);
+    CHK_STATUS(hashTableGet(pKvsPeerConnection->pDataChannels, pRtcDataChannelStats->dataChannelIdentifier, (PUINT64) &pKvsDataChannel));
```

Comment:
This will fail if the element is missing. I am OK to return this error. Alternatively, you could check for presence and return a different error if you need

Reply:
Failing would not be catastrophic based on how the application deals with it. So, it is OK if the element is missing.